README.md

```diff
@@ -97,7 +97,7 @@ This documentation is split into parts that help you through the process
     [**Wire up the matrix to your Pi**](./wiring.md). This document describes
     what goes where. You might also be interested
     in [breakout boards](./adapter) for that.
-    If you have an [Adafruit HAT], you can choose that with
+    If you have an [Adafruit HAT] or [Adafruit Bonnet](https://www.adafruit.com/product/3211), you can choose that with
```

Comment:
You don't need the link in parenthesis after that square bracket as you have defined the link in the bottom section of the README.

Reply:
Done!